cardinal/server/health.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import (
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware/untyped"
+)
+
+type HealthResponse struct {
+	IsServerRunning   bool `json:"is_server_running"`
+	IsGameLoopRunning bool `json:"is_game_loop_running"`
+}
+
+func (handler *Handler) registerHealthHandlerSwagger(api *untyped.API) error {
+	healthHandler := runtime.OperationHandlerFunc(func(params interface{}) (interface{}, error) {
```

Comment:
why not just inline this to:
api.RegisterOperation("GET", "/health", runtime.OperationHandlerFunc(...)

Reply:
it's following the pattern as all the previous endpoints. If we add more logic to this endpoint this will be the better format imo.